org.eclipse.nebula.widgets.nattable.examples.e4.rcp.feature/feature.xml

```diff
@@ -457,34 +443,6 @@
          fragment="true"
          unpack="false"/>
 
-   <plugin
```

Comment:
Do the extended xml apis come with some other feature? Because here you only remove the single bundles but I don't see the extended xml apis added.

Reply:
The platform has moved entirely away from including 3rd party bundles in features.  It's not necessary and generally problematic because it locks in a very specific version preventing updates and causing duplicates when someone else does the same but with a different version.

See no includes of batik any any feature:

![image](https://github.com/eclipse-nattable/nattable/assets/208716/d8c2e25c-2939-4a9b-9b72-41a2dc21e91a)

No inject either:

![image](https://github.com/eclipse-nattable/nattable/assets/208716/db95c989-742a-4dc5-abb6-67bfb780c3a6)

So my suggestion would be to prune down the includes as much as possible so that it doesn't lock in specific versions of anything except your own bundles...

Reply:
Doesn't mix up what the IDE does and what Tycho does.  In any case, it's quick to see what's in the product:

![image](https://github.com/eclipse-nattable/nattable/assets/208716/62e94acf-d9de-4a19-a10d-33aaebf16eb1)

Reply:
You should keep in mind that every time you include a feature a bundle that as not your own, you're feature cannot be easily installed in general when there are other versions of that bundle installed.  That's especially true for singleton bundles.

Reply:
I know. :) But as I said, that feature was never meant to be used by anyone else than the NatTable Examples Application. Interestingly your screenshot shows only half of the bundles that are really included. All the Eclipse Platform bundles are not included. If I check the folder directly there are twice as much bundles.

But again, nevermind, I will check and verify it myself. I see that the product archive files are now > 10MB bigger, but com.ibm.icu is not included anymore. So not sure where the additional size is coming from.

Anyhow your changes seem to simplify the example product setup, and maybe I am able to simplify it even more.